workers/Dockerfile

```diff
@@ -16,6 +16,11 @@ COPY data_models/dist/data-refinery-models-* data_models/
 # Get the latest version from the dist directory.
 RUN pip install data_models/$(ls data_models -1 | tail -1)
 
+COPY common/dist/data-refinery-common-* common/
+
+# Get the latest version from the dist directory.
+RUN pip install common/$(ls common -1 | tail -1)
```

Comment:
Same comment here and in `workers/Dockerfile.tests` as above (about getting the latest version).